src/angular/components/register.component.ts

```diff
@@ -12,6 +12,7 @@ import { PlatformUtilsService } from '../../abstractions/platformUtils.service';
 import { StateService } from '../../abstractions/state.service';
 
 import { KdfType } from '../../enums/kdfType';
+import { ReferenceEventRequest } from '../../models/request/referenceEventRequest';
```

Comment:
you'll notice i try to keep all of these imports in a logical grouping. all request models are together, all services together, enums together, etc. move this up with the other request models.

Reply:
Done, sorry, I missed this one.